src/Value/Size.php

```diff
@@ -13,7 +13,7 @@
 class Size extends PrimitiveValue
 {
     /**
-     * vh/vw/vm(ax)/vmin/rem are absolute insofar as they don’t scale to the immediate parent (only the viewport)
+     * vh/vw/vm(ax)/vmin/rem/svh/lvh/dvh/svw/lvw/dvw are absolute insofar as they don’t scale to the immediate parent (only the viewport)
```

Comment:
Rather than having to maintain the llist of units in the comment, it might be better to make the comment generic.
```suggestion
     * These are absolute insofar as they don’t scale to the immediate parent (only the viewport).
```